src/middleware/empty-response.ts

```diff
@@ -0,0 +1,16 @@
+import { Context, Middleware, Next } from 'koa';
+
+export default (): Middleware => (
+  async ({ response }: Context, next: Next): Promise<void> => {
+    await next();
+
+    if (response.body !== undefined) {
```

Comment:
isn't it clearer with a positive check? I get early returns, but the special case (undefined body) is here put into the main part of the function rather than the body of this `if`